tools/test-tools/pnpm-lock.yaml

```diff
@@ -13,10 +13,10 @@ importers:
         version: 2.0.3
       '@fluidframework/build-tools':
         specifier: ^0.53.0
-        version: 0.51.0(@types/node@18.19.31)
+        version: 0.53.0(@types/node@18.19.31)
```

Comment:
... how could this have been here with `specifier: ^0.53.0`? 🤔 . Feels like a merge conflict that was resolved badly in the past, but I'm skeptical we would even have merge conflicts for this file. Interesting, but maybe not worth investigating.